tests/client/amo/components/TestAddonDetail.js

```diff
@@ -223,7 +228,31 @@ describe('AddonDetail', () => {
     assert.include(src, 'image/png');
   });
 
-  it('renders a theme preview image', () => {
+  it('renders a theme preview as an img before mounting', () => {
+    const root = render({
+      addon: {
+        ...fakeAddon,
+        type: THEME_TYPE,
+        previewURL: 'https://amo/preview.png',
+      },
+      getBrowserThemeData: () => '{}',
+    });
+    const rootNode = findDOMNode(root);
+    root.setState({ mounted: false });
+
+    let image = rootNode.querySelector('.AddonDetail-theme-header-image');
+    assert.equal(image.tagName, 'IMG');
+    assert.ok(image.classList.contains('AddonDetail-theme-header-image'));
+    assert.equal(image.src, 'https://amo/preview.png');
+    assert.equal(image.alt, 'Press to preview');
+
+    root.componentDidMount();
```

Comment:
Yeah, it's a bit odd. I wanted to test that `componentDidMount()` is what's switching it though. I'll pull this out and add a test that just looks at `mounted`.